velox/exec/Task.h

```diff
@@ -344,6 +361,11 @@ class Task : public std::enable_shared_from_this<Task> {
       uint32_t splitGroupId,
       const std::vector<core::PlanNodeId>& planNodeIds);
 
+  // Adds Customized Hash Joinbridge for all the specified plan node IDs.
+  void addCustomizedJoinBridgesLocked(
+      uint32_t splitGroupId,
+      const std::vector<std::shared_ptr<const core::PlanNode>> planNodes);
```

Comment:
pass planNodes by reference to avoid copy

Reply:
Fixed